dask_cuda/utils.py

```diff
@@ -16,6 +16,19 @@ def setup(self, worker=None):
         os.sched_setaffinity(0, self.cores)
 
 
+class RMMPool:
+    def __init__(self, nbytes):
+        self.nbytes = nbytes
+
+    def setup(self, worker=None):
+        if self.nbytes is not None:
+            import rmm
+
+            rmm.reinitialize(
+                pool_allocator=True, managed_memory=False, initial_pool_size=self.nbytes
```

Comment:
I think you already said this. So apologies if I'm rehashing things. dask-cuda sets up device `0` to be the device used by the worker right? By default this is what `rmm` will use. If not, we might need to specify it.

Reply:
That's right, the relevant GPU will always be GPU 0 for every dask-cuda worker, so we don't need to specify it.